spec/integration/blocked_hotlinked_image_spec.rb

```diff
@@ -0,0 +1,138 @@
+# frozen_string_literal: true
+
+describe "hotlinked image blocking" do
```

Comment:
For the file name and description here, should it be `media` instead of `image` since we're testing for all forms of media here.